client/landing/subscriptions/components/site-subscription-page/site-subscription-page.tsx

```diff
@@ -168,18 +156,18 @@ const SiteSubscriptionPage = () => {
 						<>
 							<SiteSubscriptionSettings
 								blogId={ blogId }
-								notifyMeOfNewPosts={ notifyMeOfNewPosts }
-								emailMeNewPosts={ emailMeNewPosts }
-								deliveryFrequency={ deliveryFrequency }
-								emailMeNewComments={ emailMeNewComments }
+								notifyMeOfNewPosts={ data.delivery_methods?.notification?.send_posts }
```

Comment:
Take note that API is returning `notifications` which will be fixed in another patch. And `notification` object doesn't exist for external user, so `?` optional operator is still required.